Sources/SwiftGRPCNIO/ClientCalls/ClientCall.swift

```diff
@@ -54,6 +49,14 @@ public protocol ClientCall {
   func cancel()
 }
 
+/// Base protocol for a client call to a gRPC service.
```

Comment:
Duplication of documentation; should this be explained slightly different from `UntypedClientCall`?

Reply:
Reverted this change in favour of using generics in the test.